action.yml

```diff
@@ -12,7 +12,7 @@ runs:
   using: "composite"
   steps:
     - name: Run Bundle Audit
-      uses: planningcenter/bundle-audit-action@v1.1.0
+      uses: planningcenter/bundle-audit-action@v1.1.1
```

Comment:
Let's actually just pin this to [`v1.1`](https://github.com/planningcenter/bundle-audit-action/tree/v1.1). That should include your changes and also not require us to bump this for any other patches.